KB articles for PDF server proxy, Grid row colors, and Next.js theme switcher

WissamProgress opened the pull request:
Related to https://github.com/telerik/kendo-react-private/issues/3762

vveesseelliinnaa reviewed (approved):
Sami, the Kbs are great! It would be better, though, to include runnable sample previews along with the code snippets, so the clients can test then directly.

WissamProgress commented:
@vveesseelliinnaa For the theme switcher, I added a link to a StackBlitz example with a Next.js setup at the end since this can not be done with our snippets.

While for the Express.js example, this requires the creation of two projects a client app for the PDFExport and and an Express.js project for handling the proxy. The only thing I can think of is creating a GitHub repo, but the setup should be easy to implement by a simple copy-paste.

For the Grid styling, it is enough to use the 4 CSS styles I added in the snippet, but I agree that it's still better to have the changes visually appear to the user. I'll add a demo for it.
